src/plugins/default/markbind-plugin-plantuml.js

```diff
@@ -103,14 +84,21 @@ module.exports = {
     $('puml').each((i, tag) => {
       // eslint-disable-next-line no-param-reassign
       tag.name = 'pic';
-      const { src, cwf } = tag.attribs;
+      const { cwf } = tag.attribs;
+      const pumlContent = pluginUtil.getPluginContent($, tag, cwf);
+
+      const filePath = pluginUtil.getFilePathForPlugin(tag.attribs, pumlContent);
+
       // eslint-disable-next-line no-param-reassign
```

Comment:
I think this and the above one could this be deleted - there was a PR a while back that changed this